projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -0,0 +1,38 @@
+import { standardUser } from '../../sample-data/shared-users';
+import { generateMail, randomString } from '../user';
+import { clickHamburger } from '../checkout-flow';
+
+export function loginRegisteredUser() {
+  standardUser.registrationData.email = generateMail(randomString(), true);
+  cy.requireLoggedIn(standardUser);
+  cy.reload();
+}
+
+export function clickMyAccountMenuOption(){
+  cy.visit('/');
+  cy.get('#cx-header', { timeout: 10000 }).should('be.visible');
+  cy.onMobile(() => {
+    clickHamburger();
+  });
+  cy.get('.accNavComponent button').click();
+}
+
+export function clickCustomerSupportMenuOption(){
+  cy.get('.accNavComponent li:nth-child(14)').should('contain.text', 'Customer Service');
+  cy.get('.accNavComponent li:nth-child(14) a').click();
```

Comment:
If customer support option is in the 14th position can we change the magic number?
const CUSTOMER_SUPPORT_OPTION = 14
```suggestion
  cy.get(`.accNavComponent li:nth-child(${CUSTOMER_SUPPORT_OPTION})`).should('contain.text', 'Customer Service');
  cy.get(`.accNavComponent li:nth-child(${CUSTOMER_SUPPORT_OPTION}) a`).click();
```

Reply:
Done.